### PR TITLE
feat: add interview feedback template

### PR DESCRIPTION
**Pull request by aaliakseyenka**
#### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 💡 Background and solution

This describes the new feedback template for technical screening interview/

### CODE REVIEW

File: client/src/data/interviews/technical-screening.tsx [hunk 1]
```diff
@@ -0,0 +1,486 @@
...
+    {
+      id: 'englishCertificate',
+      type: InputType.RadioButton,
+      required: true,
+      title: 'Certified level of English',
+      description: 'Make a mark, if the student has a certificate, proving his English level',
+      options: [
+        { id: 'none', title: 'No certificate' },
+        { id: 'A1', title: 'A1' },
+        { id: 'B1', title: 'B1' },
```
Comment (valerydluski) [inline client/src/data/interviews/technical-screening.tsx:389]: { id: 'A2', title: 'A2' },

Comment (aaliakseyenka) [reply client/src/data/interviews/technical-screening.tsx:390]: thank you Valery for catching.

File: client/src/data/interviews/technical-screening.tsx [hunk 2]
```diff
@@ -0,0 +1,488 @@
...
+  title: 'Practice',
+  stepperDescription: 'Propose technical tasks to solve',
+  description: (
+    <>
+      Ask the student to solve the coding problem. See the list of examples of coding tasks or suggest another problem
+      of the same level.
+      <div>
+        <ClockCircleOutlined />
+        &nbsp; 10-30 min
+      </div>
```
Comment (valerydluski) [inline client/src/data/interviews/technical-screening.tsx:343]: Maybe move it to a renderer function or component?
```
const : React.FC<{ minutes: string }> = ({ minutes }) => (
  <div>
    <ClockCircleOutlined />
    &nbsp; {minutes} min
  </div>
);
```

**Review by valerydluski — APPROVED** [review]

File: client/src/data/interviews/technical-screening.tsx [hunk 3]
```diff
@@ -0,0 +1,488 @@
...
+export const SKILLS_LEVELS = [
+  `Doesn't know`,
+  `Poor knowledge (almost doesn't know)`,
+  'Knows something (with tips)',
+  'Good knowledge (makes not critical mistakes)',
+  'Great knowledge',
+];
+
+export const CODING_LEVELS = [
+  `Isn't able to coding`,
```
Comment (I-vasilich-I) [inline client/src/data/interviews/technical-screening.tsx:153]:
```suggestion
  `Isn't able to code`,
```

File: client/src/data/interviews/technical-screening.tsx [hunk 4]
```diff
@@ -0,0 +1,488 @@
...
+      id: 'isGoodCandidate',
+      type: InputType.Checkbox,
+      title:
+        'In your opinion, is this student a good candidate for mentoring with active interest  and motivation? Make a mark',
+      options: [{ id: 'true', title: 'The student is a good candidate for mentoring.' }],
+    },
+    {
+      id: 'decision',
+      type: InputType.Radio,
+      title: 'Do you want to mentor this student and take them in your group?',
```
Comment (I-vasilich-I) [inline client/src/data/interviews/technical-screening.tsx:453]: him/her/them :)

File: client/src/data/interviews/technical-screening.tsx (hunk 2 above)
Comment (aaliakseyenka) [reply client/src/data/interviews/technical-screening.tsx]: sure, thanks.

File: client/src/data/interviews/technical-screening.tsx (hunk 3 above)
Comment (aaliakseyenka) [reply client/src/data/interviews/technical-screening.tsx]: updated, thx

File: client/src/data/interviews/technical-screening.tsx [hunk 5]
```diff
@@ -0,0 +1,488 @@
...
+  },
+  {
+    id: 'oop',
+    title: 'OOP (Encapsulation, Polymorphism, and Inheritance)',
+    topic: 'Computer Science',
+  },
+  {
+    id: 'html',
+    title:
+      "Position and display attributes' values, tags, weight of selectors, pseudo-classes and elements, box model, relative and absolute values, em vs rem, semantic, semantic tags, etc.",
```
[the remark this answers is not in the document]
Comment (aaliakseyenka) [reply client/src/data/interviews/technical-screening.tsx]: thx, updated

File: client/src/data/interviews/technical-screening.tsx [hunk 6]
```diff
@@ -0,0 +1,488 @@
...
+        &nbsp; 5 min
+      </div>
+    </>
+  ),
+  items: [
+    {
+      id: 'finalScore',
+      type: InputType.Input,
+      title: 'Final Score',
+      description: 'We calculated average based on your marks, but you can adjust the final score',
```
[the remark this answers is not in the document]
Comment (aaliakseyenka) [reply client/src/data/interviews/technical-screening.tsx]: sure

File: client/src/data/interviews/technical-screening.tsx (hunk 4 above)
Comment (aaliakseyenka) [reply client/src/data/interviews/technical-screening.tsx:447]: AFAIK `them` is more polite here

**Pull request merged**